src/main/java/me/totalfreedom/totalfreedommod/rank/Title.java

```diff
@@ -7,6 +7,7 @@
 {
 
     DEVELOPER("a", "Developer", ChatColor.DARK_PURPLE, "Dev"),
+    FOUNDER("the", "Founder", ChatColor.DARK_RED, "TF Founder"),
```

Comment:
I know that, but no other server cares who the founder of TF is, nor the owner. Thus this should be generic enough that it can be set in the config who the founder of the server TFM is running on is